### PR TITLE
Translate "Home" into Thai

### LINKED ISSUE #21: [Home] Translate text to Thai

### PR DESCRIPTION
**Pull request by Sopaphorn**
## ✅ DoD

- [x] All work is complete
- [x] Migrations: NA
- [x] Documentation: NA
- [x] Release notes: NA
- [x] Issues linked: resolves #21

## 🛑 Problems

- Change to Thai the meaning may not be 100% correct

## 📝 Summary

- Add Thai on the English text.

## 💡 More ideas

- we should have more than one picture
- may be design to have multiple images in one

## 📸 Screenshots
Screen Shot 2564-04-20 at 16 28 20

### CODE REVIEW

File: hosting/src/views/Home/Home.html [hunk 1]
```diff
@@ -114,6 +116,16 @@
         you have an idea for an event but need help organising then we also
         welcome your submissions.
       </p>
+      <p
+        class="lg:max-w-xl text-gray-500 text-base sm:text-lg md:text-xl mt-3 sm:mt-5 md:mt-5"
+      >
+        Phits Tech มุ่งหวังที่จะรวบรวมชุมชนเทคโนโลยีทั้งในและรอบๆพิษณุโลก
```
Comment (charles-allen) [inline hosting/src/views/Home/Home.html]: At the moment the Thai text is after the English text. I think we should be a Thai-first community! **Please move the Thai text above the English text.**

**Review by charles-allen — CHANGES_REQUESTED** [review]
Please fix comments!

**Comment by Sopaphorn** [comment]
Move the Thai text above the English text.
Screen Shot 2564-04-20 at 16 28 20

**Review by charles-allen — APPROVED** [review]
It's beautiful !!

**Pull request merged**